src/EVM.hs

```diff
@@ -1075,7 +1079,9 @@ exec1 = do
           case stk of
             (x:xs) ->
               burn g_mid $ forceConcrete x "JUMP: symbolic jumpdest" $ \x' ->
-                checkJump x' xs
```

Comment:
why remove the `checkJump` here?

Reply:
it's still there but guarded under the case

Reply:
oh of course... :woman_facepalming: